src/api/sentiment.js

```diff
@@ -1,75 +1,80 @@
-const Twit = require('twit')
-const config = require('../config')
-
-const bot = new Twit(config.twitterKeys)
-
 const quote = require('positivity-api')
 
-const db = require('../helpers/db')
-
 // sentiment deps
 const unified = require('unified')
 const sentiment = require('retext-sentiment')
 const english = require('retext-english')
 
+const bot = require('../twitBot')
+
+const db = require('../helpers/db')
+
 // event is passed in from calling bot.js function
 const sentimentBot = event => {
-  // console.log('====================')
-  // console.log(event)
-  // console.log('====================')
   const processor = unified()
     .use(english)
     .use(sentiment)
   const tree = processor.parse(event.text)
 
   processor.run(tree, event.text)
 
+  // eslint-disable-next-line no-console

```

Comment:
Id there a reason for commenting each line with `// eslint-disable-next-line no-console` instead of `// eslint-disable` and `// eslint-enable` for the whole block?

Reply:
Yes we can do the block here I keep it this way because we should make sure eqch console log are for the app purpose and not just console log here for development. This eay ee make sure this is not a console log who is suppose to be here